src/main/java/org/alliancegenome/curation_api/services/validation/base/AuditedObjectValidator.java

```diff
@@ -41,21 +42,25 @@ public E validateAuditedObjectFields(E uiEntity, E dbEntity, Boolean newEntity)
 			dbEntity.setDateCreated(uiEntity.getDateCreated());
 		}
 
+		String oktaEmail = authenticatedPerson.getOktaEmail();
 		if (uiEntity.getCreatedBy() != null) {
 			Person createdBy = personService.fetchByUniqueIdOrCreate(uiEntity.getCreatedBy().getUniqueId());
 			createdBy.getEmails().size();
 			createdBy.getOldEmails().size();
 			dbEntity.setCreatedBy(createdBy);
 		} else if (newEntity) {
-			Person createdBy = personService.findPersonByOktaEmail(authenticatedPerson.getOktaEmail());
-			createdBy.getEmails().size();
-			createdBy.getOldEmails().size();
-			dbEntity.setCreatedBy(createdBy);
+			if (oktaEmail != null) {
+				Person createdBy = personService.findPersonByOktaEmail(oktaEmail);
+				createdBy.getEmails().size();
+				createdBy.getOldEmails().size();
+				dbEntity.setCreatedBy(createdBy);
+			}
 		}
 
-		Person updatedBy = personService.findPersonByOktaEmail(authenticatedPerson.getOktaEmail());
-		dbEntity.setUpdatedBy(updatedBy);
-
+		if (oktaEmail != null) {
+			Person updatedBy = personService.findPersonByOktaEmail(authenticatedPerson.getOktaEmail());
+			dbEntity.setUpdatedBy(updatedBy);
+		}
```

Comment:
authenticatedPerson is the Person object from the database. You shouldn't have to relookup the person just dbEntity.setUpdateBy(authenticatedPerson);